packages/adaptive-web-components/src/components/number-field/number-field.template.ts

```diff
@@ -44,6 +44,7 @@ export const NumberFieldAnatomy: ComponentAnatomy<typeof NumberFieldConditions,
     interactivity: Interactivity.disabledAttribute,
     conditions: NumberFieldConditions,
     parts: NumberFieldParts,
+    focus: Focus.partWithin("root", "control"), // How to make this `NumberFieldParts.root` instead of a string?
```

Comment:
This comment in code is related to the GitHub comment I posted in `combobox.template.ts`. The goal was to be able to find breaks in the styling when a component element structure changes. Maybe just a typed list of strings is fine.